lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -25,7 +25,7 @@ def as_json
           meta = meta_for(association)
           hash[:meta] = meta if meta
```

Comment:
I've force pushed this change. I still find the updated code less descriptive, but my salience is very low.